Package/PackageActions/List.cs

```diff
@@ -73,6 +74,8 @@ protected override int LockedExecute(CancellationToken cancellationToken)
                 Target = FileSystemHelper.GetCurrentInstallationDirectory();
 
             HashSet<PackageDef> installed = new Installation(Target).GetPackages().ToHashSet();
+            if (checkOs)
+                installed = installed.Where(p => p.OS.Split(',').Any(o => o.Equals(OS, StringComparison.OrdinalIgnoreCase))).ToHashSet();
```

Comment:
This looks good, but I think we should try to make the code a bit less implicit

by doing something like
```cs
static bool SupportsOS(this PackageDef pkg, string os) => pkg.OS.Split(',').Any(o => o.Equals(os, StringComparison.OrdinalIgnoreCase));

installed = installed.Where(pkg =>pkg.SupportsOS(OS)).ToHashSet();
```
```

Reply:
I agree.